### PR TITLE
Remove ROOT application

### PR DESCRIPTION
**Pull request by mesmith75**
Removes ROOT RTHandlers from LHCb.

### CODE REVIEW

**Review by alexanderrichards — DISMISSED** [review]

**Review by egede — COMMENTED** [review]
Rather than just removing the ROOT application from DIRAC, should we remove it everywhere?

**Review by egede — DISMISSED** [review]
I guess you will discover if there are some test cases that should be removed.

File: ganga/GangaCore/Lib/Executable/Executable.py [hunk 1]
```diff
@@ -288,3 +288,28 @@ def prepare(self, app, appconfig, appmasterconfig, jobmasterconfig):
...
+    ascii_length = len(ascii_uppercase)
+    sample = []
+    for _ in range(6):
+        addToSample(sample, ascii_length)
+    assert(len(sample) == 6)
+
+    # seed is set to clock during import
+    return ''.join([str(a) for a in sample])
+
+
```
Comment (alexanderrichards) [inline ganga/GangaCore/Lib/Executable/Executable.py:315]: Out of context it seems strange that this was even in the ROOT module. Makes me wonder if it was only used in said module and maybe a candidate for removing altogether. Do you know where this is used?

**Review by alexanderrichards — APPROVED** [review]

File: ganga/GangaCore/Lib/Executable/Executable.py [hunk 2]
```diff
@@ -288,3 +288,28 @@ def prepare(self, app, appconfig, appmasterconfig, jobmasterconfig):
 allHandlers.add('Executable', 'CREAM', LCGRTHandler)
 allHandlers.add('Executable', 'ARC', LCGRTHandler)
 allHandlers.add('Executable', 'Slurm', RTHandler)
+
+def randomString():
```
Comment (alexanderrichards) [inline ganga/GangaCore/Lib/Executable/Executable.py:292]: *I realised the previous comment didn't really show the whole code so more useful to see the function prototype*

**Review by alexanderrichards — APPROVED** [review]

File: ganga/GangaCore/Lib/Executable/Executable.py (hunk 1 above)
Comment (alexanderrichards) [reply ganga/GangaCore/Lib/Executable/Executable.py:315]: I'm referring to the `randomString` function which unfortunately didn't show here in it's entirety.

Comment (mesmith75) [reply ganga/GangaCore/Lib/Executable/Executable.py:315]: Yes I agree it was weird. Would it be better put elsewhere?

Comment (alexanderrichards) [reply ganga/GangaCore/Lib/Executable/Executable.py:315]: for now, putting it in `Executable` seems as good as anything. My point was more that if it was only used in the now defunct `Root` module then maybe we don't need it at all?

Comment (mesmith75) [reply ganga/GangaCore/Lib/Executable/Executable.py:315]: Ah, It seems to be used in various places so I kept it.

**Pull request merged**